db/fixtures/buzzes.yml

```
@@ -1,2 +1,49 @@
+  published_on: 2025-06-30
+
+buzz4:
+  title: "🎉 フィヨルドブートキャンプ100日目の振り返り - Kimmy's Notes"
+  url: "https://kimmy-2ka.hatenablog.com/entry/2025/06/28/094516"
+  published_on: 2025-06-28
+
+buzz6:
+  title: "オフラインイベント「ゆるもく交流会」を初主催しました - IT's not Rocket Science!"
+  url: "https://hiroblogdesu.hatenablog.com/entry/2025/06/18/021242"
+  published_on: 2025-06-17
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)